### PR TITLE
Item Upload using REST API

### PR DESCRIPTION
**Pull request by officialpranav**
i forgot to do commits lol

### CODE REVIEW

File: assets/common.js [hunk 1]
```diff
@@ -163,4 +167,149 @@ function loadTheme(){
...
+    // let response = await fetch(url, {
+    //     method: "POST",
+    //     headers: {
+    //         "apiKey":apiKey,
+    //         "filetype":"XML",
+    //     },
+    //     body: xmldoc,
+    // });
+    // console.log(await response.json());
+    // console.log("Success");
```
Comment (jonathanmajh) [inline assets/common.js:303]: remove this

File: renderer/item_main.js [hunk 2]
```diff
@@ -3,11 +3,13 @@ const { clipboard, ipcRenderer, shell } = require('electron');
 const Database = require('../assets/indexDB');
 const Validate = require('../assets/validators');
 
+const testItem = new Item(undefined,"test","EA","401","ERM","sdffas");
```
Comment (jonathanmajh) [inline renderer/item_main.js:6]: can be removed?

Comment (officialpranav) [reply renderer/item_main.js:6]: ye

File: renderer/worker.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ const TranslationDatabase = require('../assets/item_translation/item-translation
 const path = require('path');
 const Translation = require('../assets/item_translation/item-translation');
 const fs = require('fs');
-const { debug } = require('console');
+const { debug, error } = require('console');
```
Comment (jonathanmajh) [inline renderer/worker.js:13]: can be removed?

**Pull request merged**